mautic/mautic
# FEAT: Adds parsing of {pagelink=\d} token to form redirect action

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🟢 
| Related user documentation PR URL      | https://github.com/mautic/user-documentation/pull/316
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 


## Description

This PR adds parsing of `{pagelink=$PAGE_ID}` token to the redirect form submit action.

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a landing page, if necessary. Note the page ID.
3. Create a new or edit an existing form
4. Set the submit action to redirect
5. Enter the page token `{pagelink=$PAGE_ID}`. Replace `$PAGE_ID` by the page's actual ID
6. Load your form in a browser session where you're not logged into the Mautic backend
7. Submit the form
8. See that Mautic redirects you to the correct landing page.
